hardware/OTGWBase.cpp

```diff
@@ -396,11 +396,6 @@ void OTGWBase::ParseLine()
 		int idx=0;
 		bool bIsFirmware5 = (results.size()==34);
 
-		std::string sFirmwareMsg = "pre version 5 firmware";
-		if (bIsFirmware5)
-			sFirmwareMsg = "firmware 5 or higher";
-		Debug(DEBUG_HARDWARE, "OTGW: Assuming %s! Parsing the following PS=1 input line : .%s.", sFirmwareMsg.c_str(), sLine.c_str());
-
```

Comment:
@gizmocuz , Is there a specific reason to remove this? As this message uses a specific Debug loglevel (hardware) I thought it makes sense to have something in there as it helps OTGW users/owners to get specific insights in the PS=1 messages from their OTGW module if they really wanted to. Something they otherwise can not see unless hooking up for example the specific OTmonitor software.

These message do not appear in the normal (debug) logging anyway.

(Just to understand the reasoning as I thought the extra debuglevels where there to make it both easier to debug but also much more specific and not flooding normal (debug)logging?)

Reply:
@kiddigital , I run in hardware debug mode, and after seeing this message 100 times, I know it's v4 or v5.
If you want to display this information for a specific reason, I would add a boolean (m_bDisplayedVersion), default false, and display the firmware version 1 time if the boolean is false, and set the boolean to true

Reply:
Also, although it ain't logged, the message is still created and send to the log function

Reply:
Ok clear. Agree that the versioning info is not relevant to keep repeating. Doing it once at start is indeed sufficient. But now also the PS=1 line info is not displayed when in (certain) debug mode. Isn't that the relevant info that you would like to see during debugging (as every reading contains the actual readings)? So you can see/find odd readings, find out if things have changes, etc.

For me it is fine right now, let's see if there is demand for additional debug info from Domoticz and than we can always add something :)

(The desire to buying an OTGW myself has been bumped up a lot on my wish list because of working on this ;) )

Reply:
Let's see if someone needs this information.
Love my OTGW (connected via ser2net on a TPLink nano

Reply:
> Let's see if someone needs this information.
> Love my OTGW (connected via ser2net on a TPLink nano

Same kind of setup here. What should/can I do to assist ? 
Received a new boiler yesterday.

Reply:
@sincze , I think all is fine and you should be able to use the latest beta version with your OTGW and also when you upgrade your OTGW to firmware v5